diff --git a/creature.cpp b/creature.cpp
index 71f7450..9a4fcd8 100755
--- a/creature.cpp
+++ b/creature.cpp
@@ -6300,6 +6300,8 @@ MOB::pageCharacterDump(bool ondeath, bool didwin, int truetime)
 	gfx_pager_addtext("iPOWDER Version");
 #elif defined(LINUX)
 	gfx_pager_addtext("Linux Version");
+#elif defined(USING_WII)
+	gfx_pager_addtext("Wii Version");
 #else
 	gfx_pager_addtext("Unknown SDL Version");
 #endif
